lib/feedback_form.dart

```diff
@@ -39,30 +40,30 @@ class _FeedbackFormState extends State<FeedbackForm> {
   @override
   Widget build(BuildContext context) {
     return Container(
-            padding: EdgeInsets.all(15),
-            child: Column(
-              children: [
-                TextField(
-                  controller: controller,
-                  decoration: InputDecoration(
-                      hintText: 'Your feedback message', labelText: 'Feedback'),
-                ),
-                SwitchListTile(
-                  title: Text('Anonymous feedback'),
-                  value: isAnonymous,
-                  onChanged: (value) {
-                    setState(() {
-                      isAnonymous = value;
-                    });
-                  },
-                ),
-                RaisedButton(
-                  child: Text('Submit Feedback'),
-                  onPressed: handleSubmitFeedback,
-                )
-              ],
-            ),
-          );
+      padding: EdgeInsets.all(15),
```

Comment:
These are changes based on spacing only? It should not be here. We should agree on a spacing/tabbing style. Please remove this to avoid confusion.

Reply:
Yea, we used to use some kind of linters🙂 okay, I don't sure how, but I'll try to remove ( or just leave it here, because another merge conflict will cover this file anyways)